packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -280,6 +287,16 @@ export class ChatContext {
   }
 
   public setChatMessage(threadId: string, message: ChatMessageWithStatus): void {
+    /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
```

Comment:
Maybe break this out into a separate method for simplicity of CC and clarity of functionality?

Reply:
Refactored